Simplified/NYPLMyBooksViewController.h

```diff
@@ -5,6 +5,8 @@
 - (id)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil NS_UNAVAILABLE;
 
+- (void)didSelectSync;
```

Comment:
A couple of things on this one:

1. Methods with names like `didSelectSync` are generally called in response to an action attached to an interface control. I was a little surprised to see a method with this name exposed in the header. If we want to be able to tell My Books to sync, perhaps we can expose a method named `sync` instead which just calls `didSelectSync` internally?

2. In the original design, My Books would sync automatically whenever `NYPLBookRegistryDidChangeNotification` was fired. Is there a way we can go back to having My Books completely manage its own syncing again so we don't need to worry so much about actively keeping everything up-to-date as appropriate?